description: fix ambiguous namespace alias errors that occur in jumbo build
author: Michael Gilbert <mgilbert@debian.org>

--- a/extensions/common/manifest_handlers/permissions_parser.cc
+++ b/extensions/common/manifest_handlers/permissions_parser.cc
@@ -27,13 +27,13 @@
 #include "extensions/common/url_pattern_set.h"
 #include "url/url_constants.h"
 
+namespace keys = extensions::manifest_keys;
+namespace errors = extensions::manifest_errors;
+
 namespace extensions {
 
 namespace {
 
-namespace keys = manifest_keys;
-namespace errors = manifest_errors;
-
 struct ManifestPermissions : public Extension::ManifestData {
   ManifestPermissions(std::unique_ptr<const PermissionSet> permissions);
   ~ManifestPermissions() override;
--- a/extensions/common/manifest_handlers/requirements_info.cc
+++ b/extensions/common/manifest_handlers/requirements_info.cc
@@ -12,10 +12,10 @@
 #include "extensions/common/error_utils.h"
 #include "extensions/common/manifest_constants.h"
 
-namespace extensions {
+namespace keys = extensions::manifest_keys;
+namespace errors = extensions::manifest_errors;
 
-namespace keys = manifest_keys;
-namespace errors = manifest_errors;
+namespace extensions {
 
 RequirementsInfo::RequirementsInfo(const Manifest* manifest)
     : webgl(false), window_shape(false) {}
--- a/extensions/common/manifest_handlers/sandboxed_page_info.cc
+++ b/extensions/common/manifest_handlers/sandboxed_page_info.cc
@@ -17,13 +17,13 @@
 #include "extensions/common/manifest_constants.h"
 #include "extensions/common/url_pattern.h"
 
+namespace keys = extensions::manifest_keys;
+namespace errors = extensions::manifest_errors;
+
 namespace extensions {
 
 namespace {
 
-namespace keys = extensions::manifest_keys;
-namespace errors = manifest_errors;
-
 const char kDefaultSandboxedPageContentSecurityPolicy[] =
     "sandbox allow-scripts allow-forms allow-popups allow-modals; "
     "script-src 'self' 'unsafe-inline' 'unsafe-eval'; child-src 'self';";
--- a/extensions/common/manifest_handlers/shared_module_info.cc
+++ b/extensions/common/manifest_handlers/shared_module_info.cc
@@ -22,11 +22,11 @@
 #include "extensions/common/permissions/permission_set.h"
 #include "extensions/common/permissions/permissions_data.h"
 
-namespace extensions {
+namespace keys = extensions::manifest_keys;
+namespace values = extensions::manifest_values;
+namespace errors = extensions::manifest_errors;
 
-namespace keys = manifest_keys;
-namespace values = manifest_values;
-namespace errors = manifest_errors;
+namespace extensions {
 
 namespace {
 
